packages/web3-eth-accounts/src/account.ts

```diff
@@ -410,7 +610,20 @@ export const privateKeyToAccount = (privateKey: string | Buffer): Web3Account =>
 };
 
 /**
- * Returns an acoount
+ *
+ * Generates and returns a Web3Account object that includes the private and public key
```

Comment:
As we removed entropy param in 4.x so I think lets make it more visible to users how it works now, may be add some details like following in doc:

`It uses ethereum-cryptography/secp256k1 https://www.npmjs.com/package/ethereum-cryptography#secp256k1-curve for creation of private key that is cryptographically secure random number with certain characteristics. `